Scripts/parameters/assignment.py

```diff
@@ -186,6 +186,21 @@
         "iht": 1 / 0.497,
     },
 }
+# Factor for converting weekday traffic into yearly day average
+years_average_day_factor = 0.85
+# Factor for converting day traffic into 7:00-22:00 traffic
+share_7_22_of_day = 0.9
+# Noice zone width as function of start noise
+noise_zone_width = [
+    {"threshold": 80, "function": lambda x: 225},
+    {"threshold": 77, "function": lambda x: 153 + 66/3*x},
+    {"threshold": 74, "function": lambda x: 109 + 44/3*x},
+    {"threshold": 71, "function": lambda x: 78 + 31/3*x},
+    {"threshold": 68, "function": lambda x: 57 + 21/3*x},
+    {"threshold": 65, "function": lambda x: 41 + 16/3*x},
+    {"threshold": 55, "function": lambda x: 10 + 31/10*x},
```

Comment:
Overall very nice refactoring here and in `calc_noise()` where these `function`s are being called! `lambda`s were a new thing for me, thanks! 😄